.github/workflows/release.yaml

```diff
@@ -28,6 +28,15 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+      - uses: azure/docker-login@v1
+        with:
+          login-server: index.docker.io
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Docker Publish
+        run: |
+          make docker-publish-initContainer
+          make docker-publish-kyverno
```

Comment:
Is docker image published on PR creates or merges?

Reply:
yes on release